packages/serve-sim/Sources/SimNative/build.sh

```
@@ -26,25 +24,29 @@ if [ ! -d "$PKG/node_modules/node-swift" ]; then
-  -c release \
-  --product "$PRODUCT" \
-  --package-path "$PKG" \
-  --build-path "$BUILD_DIR" \
-  -Xlinker -undefined -Xlinker dynamic_lookup
+build_arch() {
+  build_flags=(
+    -c release
+    --product "$PRODUCT"
+    --package-path "$PKG"
+    --build-path "$BUILD_DIR"
+    --arch $1
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)